istioctl/cmd/root.go

```diff
@@ -74,6 +88,33 @@ func defaultLogOptions() *log.Options {
 	return o
 }
 
+// ConfigAndEnvProcessing uses spf13/viper for overriding CLI parameters
+func ConfigAndEnvProcessing() error {
+	if IstioConfig != defaultIstioctlConfig {
+		// Warn if user incorrectly customized $ISTIOCONFIG
+		if _, err := os.Stat(IstioConfig); os.IsNotExist(err) {
+			fmt.Fprintf(os.Stderr, "Warning: Configuration file %q does not exist\n", IstioConfig)
```

Comment:
Do we want to continue or error here? I can imagine some CI/CD "silently" failing (if no one is reading the CI/CD output).

Reply:
Fixed